src/components/CardDetails/CardDetails.tsx

```diff
@@ -0,0 +1,38 @@
+import { ICardDetails } from "@data/interfaces/Card";
+import styles from "./CardDetails.module.css";
+import { Text } from "@components/Text/Text";
+import { Heading } from "@components/Heading/Heading";
+const CardDetails = ({
+  logo,
+  company,
+  quote,
+  feedback,
+  name,
+  position,
+}: ICardDetails) => {
+  return (
+    <article className={styles.cardDetails}>
+      <header className={styles.imgWrapper}>
+        <img src={logo} alt={company} />
+      </header>
+      <main className={styles.content}>
+        <blockquote className={styles.quote}>&quot;{quote}&quot;</blockquote>
+        <Text text={feedback} weight={"regular"} classList={styles.text} />
+        <div className={styles.clientInfo}>
+          <Heading
+            text={name}
+            level={1}
+            weight={"bold"}
+            classList={styles.title}
+          />
+          <Heading
+            text={`${position} at ${company}`}
+            level={2}
```

Comment:
Level refers to the heading level h1, h2, h3, and so on. 

- We know there is only one `h1` element and it is mandatory to be only one. Usually is the main title of the page, so here level 1 is wrong.
- `h2` are usually the heading of each section, looking at the design I can notice this is also wrong